### PR TITLE
Feature add sass framework

### PR DESCRIPTION
**Pull request by robinbastien**

### CODE REVIEW

File: assets/style.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+# Leave this front-matter block here :)
+---
+
+@import "main"
```
Comment (KennethAshley) [inline assets/style.scss]: I noticed there is no semi-colon. But doesn't seem like the bundled `main.scss` that is came with Jekyll has it either. Is there a reason for this?

Comment (robinbastien) [reply assets/style.scss]: Last line in a document so it builds fine, but you're right. We should be linting it. Will update :p

Comment (KennethAshley) [reply assets/style.scss]: Thanks man!

Comment (robinbastien) [reply assets/style.scss]: Yeah that's a weird file. If you take the YAML front-matter out from the top, it doesn't compile properly. Wish we didn't need this file... as all it does is points to the actual SASS dir

**Pull request merged**